meta/saiserialize.c

```diff
@@ -832,10 +851,22 @@ int sai_deserialize_ip_address(
         return res;
     }
 
-    SAI_META_LOG_WARN("failed to deserialize '%.*s' as ip address",
-            INET6_ADDRSTRLEN, buffer);
+    for (idx = 0; idx < sizeof(*ip_address); idx++)
```

Comment:
make no sense to deserialize numbers, since you have 2 types of ip address 4 and 6, other types are not supported

Reply:
See the comments in the serialize counterpart.